swarm/network/stream/streamer_test.go

```diff
@@ -1257,8 +1257,9 @@ func TestGetSubscriptionsRPC(t *testing.T) {
 		simulation.NewPeerEventsFilter().ReceivedMessages().Protocol("stream").MsgCode(subscribeMsgCode),
 	)
 
-	// upload a snapshot
-	err := sim.UploadSnapshot(fmt.Sprintf("testing/snapshot_%d.json", nodeCount))
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
+	defer cancel()
+	err := sim.UploadSnapshot(ctx, fmt.Sprintf("testing/snapshot_%d.json", nodeCount))
```

Comment:
For consistency if you extract `fmt.Sprintf` into a`filename` variable please make that here, too.

Going one step further this filename generation is a repeating pattern. The format string `"testing/snapshot_%d.json"` seems a little fragile and the pattern is unnecessary detail if I just want to understand the test. => Maybe worth a very simple method like `getSnapshotFilename(nodeCount int)`?